cupy/statistics/histogram.py

```diff
@@ -159,10 +208,38 @@ def histogram(x, bins=10, range=None, density=False):
         # TODO(unno): comparison between complex numbers is not implemented
         raise NotImplementedError('complex number is not supported')
 
+    if not isinstance(x, cupy.ndarray):
+        raise ValueError("x must be a cupy.ndarray")
+
+    x, weights = _ravel_and_check_weights(x, weights)
     bin_edges, uniform_bins = _get_bin_edges(x, bins, range)
```

Comment:
```suggestion
    bin_edges, _ = _get_bin_edges(x, bins, range)
```
uniform_bins seems to be unused?

Reply:
I have now  removed the variable from _get_bin_edges itself.